### ISSUE TITLE
Link to model from GET endpoints

### ISSUE THREAD

**Issue by marclave** [scalar/scalar#1847]
### Discussed in [https://github.com/scalar/scalar/discussions/1831](<https://github.com/scalar/scalar/discussions/1831>)

Originally posted by **yonderblue** May 22, 2024<br>Would be great to link to models. For example [here](<https://docs.scalar.com/project-default/version-default/reference-nOvFu4SFm1hCRPclrBhuv#tag/planets/get/planets/%7BplanetId%7D>) it shows an example output and optionally the schema if you hit the Show Schema checkbox, but not a link over to the Model section of Planet.

**Comment by hanspagel** [comment]
Currently, we don't know what’s referenced. We could add a feature to the parser, that adds hints to the replaced content, though. 👀

**Comment by geoffgscott** [comment]
@hanspagel I think when we bring in the workspace logic from the upcoming client we should be able to maintain a reference to the original model.

**Comment by hanspagel** [comment]
Not really. The parser replaces $ref with the actual content and the information that the content was reference is lost. We’d have to leave a hint once the content is dereferenced. 👍

**Comment by hanspagel** [comment]
Relevant: https://github.com/scalar/scalar/discussions/3811

**Comment by AdrianoAE** [comment]
Any update on this? Also this is for all endpoints not just GET

**Comment by druskus20** [comment]
Coming from using [aide](https://github.com/tamasfe/aide), which abstracts over API frontends. 

From what I gather, neither Redoc, Swagger or Scalar do this. For me this would be the decisive feature that makes me choose one over the other.

**Comment by amritk** [comment]
Yes! Once we have moved over to the new store this should  be possible

**Comment by amritk** [comment]
Alright we got the new store, now what are we looking for exactly?
